lib/osf-components/addon/components/registries/partial-registration-modal/template.hbs

```diff
@@ -1,32 +1,36 @@
-<OsfDialog @isOpen={{@modalManager.isOpen}} @renderInPlace={{@modalManager.renderInPlace}} as |dialog|>
+<OsfDialog
+    @isOpen={{@isOpen}}
+    @onClose={{@onClose}}
+    as |dialog|
+>
     <dialog.heading local-class='modal-title'>
         {{t 'registries.partialRegistrationModal.title'}}
     </dialog.heading>
     <dialog.main>
-        <OsfButton 
-            @type=='link' 
-            local-class='select-clear-button' 
-            @onClick={{action @modalManager.selectAll}}
+        <OsfButton
+            @type=='link'
```

Comment:
`=` instead of `==`

Reply:
good catch. funny this wasn't caught before.